### ISSUE TITLE
beforeEach, before, todo, only, skip etc not working with ava - is not a function

### ISSUE THREAD

**Issue by kennethlynne** [wallabyjs/public#728]
### Issue description or question
![image](https://cloud.githubusercontent.com/assets/390817/17420718/c8964088-5aa4-11e6-9526-784a46f2f270.png)

![image](https://cloud.githubusercontent.com/assets/390817/17420723/cfdeb26c-5aa4-11e6-9581-31c6e993af9e.png)

![image](https://cloud.githubusercontent.com/assets/390817/17420750/046806a0-5aa5-11e6-9ad0-31549c19860a.png)

### Wallaby.js configuration file

```javascript
module.exports = function (wallaby) {
  return {
    files: [
      'app/**/*.js',
      '!app/**/*.spec.js'
    ],

    tests: [
      'app/**/*.spec.js'
    ],

    env: {
      type: 'node'
    },

    compilers: {
      '**/*.js': wallaby.compilers.babel()
    },

    testFramework: 'ava',

    setup: function () {
      require('babel-polyfill');
    },

    debug: true
  };
};
```

### Code editor or IDE name and version
WebStorm v11

### OS name and version
OSX

**Comment by kennethlynne** [comment]
Verified in https://github.com/wallabyjs/ava-sample/blob/master/app/index.spec.js

**Comment by ArtemGovorov** [comment]
Thanks for the report, the bug is fixed in core 1.0.271, will be updated automatically, or just restart the IDE and start wallaby to force the update.
Please also check if the change fixes this issue: https://github.com/wallabyjs/public/issues/727. If not, please provide some sample for that issue.
